template/fullstack/authentication/server/config/index.js

```diff
@@ -0,0 +1,38 @@
+// We reuse this import in order to have access to the `body` property in requests
+const express = require("express");
+
+// ℹ️ Responsible for the messages you see in the terminal as requests are coming in
+// https://www.npmjs.com/package/morgan
+const logger = require("morgan");
+
+// ℹ️ Needed when we deal with cookies (we will when dealing with authentication)
+// https://www.npmjs.com/package/cookie-parser
+const cookieParser = require("cookie-parser");
+
+// ℹ️ Needed to accept from requests from 'the outside'. CORS stands for cross origin resource sharing
+// unless the request if from the same domain, by default express wont accept POST requests
```

Comment:
Minor typo: "unless the request **if** from the same domain...". Replace by `is made`